### PR TITLE
tests: cut off infinite loop

### PR DESCRIPTION
**Pull request by jarrpa**
Signed-off-by: Jose A. Rivera <jarrpa@redhat.com>

---
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/288)

### CODE REVIEW

File: tests/complex/test-inside-dynamic-provisioning.sh [hunk 1]
```diff
@@ -51,9 +51,11 @@ kubectl create -f "./${PVC}.yaml"
 
 
 echo "verifying the pvc has been created and bound"
+s=0
 PVCstatus=$(kubectl get pvc | grep "${PVC}" | awk '{print $2}')
-while [[ "$PVCstatus" != 'Bound' ]] ; do
+while [[ "$PVCstatus" != 'Bound' ]] && [[ s -lt 30 ]]; do
```
Comment (obnoxxx) [inline tests/complex/test-inside-dynamic-provisioning.sh]: `$s` ?

File: tests/complex/test-inside-dynamic-provisioning.sh [hunk 2]
```diff
@@ -51,9 +51,11 @@ kubectl create -f "./${PVC}.yaml"
...
 	sleep 1
+	s+=1
```
Comment (obnoxxx) [inline tests/complex/test-inside-dynamic-provisioning.sh]: I think that does not work:
```
$ var=1
$ var+=1
$ echo $var
11
$ 
```

**Comment by jarrpa** [comment]
@obnoxxx Oops, I think I didn't save a few changes. Pushed update.

**Comment by obnoxxx** [comment]
LGTM

(I do hope though that this somewhat arbitrary 30 retries number will  not turn out to be insufficent.)

**Pull request merged**